bittide/src/Bittide/ProcessingElement.hs

```diff
@@ -35,33 +38,36 @@ data PeConfig nBusses where
     InitialContent depthD (Bytes 4) ->
     PeConfig nBusses
 
--- | 'Contranomy' based RV32IMC core together with instruction memory, data memory and
+-- | VexRiscV based RV32IMC core together with instruction memory, data memory and
 -- 'singleMasterInterconnect'.
 processingElement ::
   forall dom nBusses .
   ( HiddenClockResetEnable dom
   , KnownNat nBusses, 2 <= nBusses, CLog 2 nBusses <= 30) =>
   PeConfig nBusses ->
   Circuit
-    ()
-    ( Vec (nBusses-2) (Wishbone dom 'Standard (MappedBus 32 nBusses) (Bytes 4))
+    (CSignal dom JtagIn)
+    ( Vec (nBusses-2) (Wishbone dom 'Standard (MappedBusAddrWidth 32 nBusses) (Bytes 4))
+    , CSignal dom JtagOut
```

Comment:
I think there is a `Protocol` implementation for `Jtag` somewhere, no? Wouldn't that work instead of having an inbound and outbound `CSignal`?